packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -164,16 +170,27 @@ def get_track_comments(args, track_id, current_user_id=None):
                 Comment.entity_type == "Track",
                 CommentThreadAlias.parent_comment_id
                 == None,  # Check if parent_comment_id is null
-                (CommentReport.comment_id == None)
-                | (current_user_id == None)
-                | (CommentReport.user_id != current_user_id),
                 or_(
-                    MutedUser.muted_user_id == None, MutedUser.is_delete == True
+                    CommentReport.comment_id == None,
+                    current_user_id == None,
+                    CommentReport.user_id != current_user_id,
+                ),
+                or_(
+                    CommentReport.comment_id == None,
+                    CommentReport.comment_id != artist_id,
+                ),
+                or_(
+                    MutedUser.muted_user_id == None,
+                    MutedUser.is_delete == True,
                 ),  # Exclude muted users' comments
             )
             .having(
                 (func.count(ReplyCountAlias.comment_id) > 0)
-                | (Comment.is_delete == False)
+                | (Comment.is_delete == False),
+            )
+            # Ensure that the combined follower count of all users who reported the comment is below the threshold.
+            .having(
+                func.coalesce(func.sum(AggregateUser.follower_count), 0) <= 1000,
```

Comment:
should be `COMMENT_REPORT_KARMA_THRESHOLD`?

Reply:
duh, yess